src/components/list-podcast/script.js

```diff
@@ -19,7 +20,7 @@ export default {
       this.$emit('add-favorite', trackId);
     },
     playTrack(trackId) {
-      this.$emit('play-track', trackId);
+      this.$root.$emit(MESSAGE_PLAY_TRACK, trackId);
```

Comment:
As before: we can **emit** the complete entity

Reply:
Improved.